tests/pdk/finfet_pdk/test_circuits.py

```diff
@@ -324,28 +324,30 @@ def test_niwc_opamp_split():
     {"constraint": "Route", "min_layer": "M2", "max_layer": "M3"},
     {"constraint": "PowerPorts", "ports": ["vccx"]},
     {"constraint": "GroundPorts", "ports": ["vssx"]},
-    {"constraint": "GroupBlocks", "instances": ["mtail"], "instance_name": "xmtail0",
+    {"constraint": "GroupBlocks", "instances": ["mtail"], "instance_name": "xmtail0", "template_name":"mtail0",
```

Comment:
@kkunal1408 , why `template_name`s are added to this test?

Reply:
template names are not necessary for the constraint. These were added to accommodate the subcircuit names in the test assert statements "assert size_tbl['DP'] == [(2,4)]". If template_name were not added, default name would be primitive_23xxxx.